plugins/palo_alto_cortex_xdr/plugin.spec.yaml

```diff
@@ -283,10 +1105,34 @@ actions:
 triggers:
   get_incidents:
     title: Get Incidents
-    description: Get Incidents
+    description: Get incidents
+    input:
+      interval:
+        title: Interval
+        description: Interval
+        type: integer
+        required: true
+        example: 5
     output:
       incident:
         title: Incident
         description: Incident
-        type: object
+        type: incident
+        required: false
+  get_alerts:
+    title: Get Alerts
+    description: Get alerts
+    input:
+      interval:
```

Comment:
Use the frequency style here https://docs.rapid7.com/insightconnect/style-guide/#triggers